camera_matrix: 
    rows: 3
    cols: 3
    dt: f
    data: [ 1412.526482, -2.900182, 1025.279538, 
            0.000000, 1410.618567, 765.485729, 
            0.000000, 0.000000, 1.000000]
distortion_coefficients: 
    rows: 1
    cols: 5
    dt: f
    data: [ -0.321503, 0.168106, 0.000001, 
            0.000282, -0.052212]

